packages/taquito-rpc/src/types.ts

```diff
@@ -581,8 +581,10 @@ export interface OperationContentsAndResultMetadataOrigination {
 export interface ConstantsResponse {
   proof_of_work_nonce_size: number;
   nonce_length: number;
-  max_revelations_per_block: number;
+  max_revelations_per_block?: number; // carthagenet
```

Comment:
@roxaneletourneau what do you think about making integration tests that fetch the constant values from a block-level for each protocol? Below is a list of block levels for protos 0 to 6.

|Protocol Hash                                         | Proto Number | Level   |
|------------------------------------------------------|--------------|---------|
|`PrihK96nBAFSxVL1GLJTVhu9YnzkMFiBeuJRPA8NwuZVZCE1L6i` | `0`          | `0`     |
|`PtCJ7pwoxe8JasnHY8YonnLYjcVHmhiARPJvqcC6VfHT5s8k8sY` | `1`          | `10000` |
|`PsYLVpVvgbLhAhoqAkMFUo6gudkJ9weNXhUYCiLDzcUpFpkk8Wt` | `2`          | `100000`|
|`PsddFKi32cMJ2qPjf43Qv5GDWLDPZb3T3bF6fLKiF5HtvHNU7aP` | `3`          | `300000`|
|`Pt24m4xiPbLDhVgVfABUjirbmda3yohdN82Sp9FeuAXJ4eV9otd` | `4`          | `600000`|
|`PsBabyM1eUXZseaJdmXFApDSBqj8YBfwELoxZHHW77EMcAbbwAS` | `5`          | `700000`|
|`PsCARTHAGazKbHtnKfLzQg3kms52kSRpgnDY982a9oYsSXRLQEb` | `6`          | `900000`|

Reply:
#449 Created this issue to implement these tests.